Gravity/Gravity.NUnit.Integration/RSAPI_IntegrationTest.cs

```diff
@@ -0,0 +1,395 @@
+                _servicesManager = helper.GetServicesManager();
+                Console.WriteLine("Services Manager Created.");
+
+                Console.WriteLine("Creating workspace.....");
+                if (!_debug)
+                {
+                    _workspaceId =
+                        Relativity.Test.Helpers.WorkspaceHelpers.CreateWorkspace.CreateWorkspaceAsync(_workspaceName,
+                            ConfigurationHelper.TEST_WORKSPACE_TEMPLATE_NAME, _servicesManager, ConfigurationHelper.ADMIN_USERNAME,
+                            ConfigurationHelper.DEFAULT_PASSWORD).Result;
+                    Console.WriteLine($"Workspace created [WorkspaceArtifactId= {_workspaceId}].....");
+                }
+                else
+                {
+                    //must point _workspaceId = valid workspace with application already installed
+                    _workspaceId = _debugWorkspaceId;
+                    Console.WriteLine($"Using existing workspace [WorkspaceArtifactId= {_workspaceId}].....");
+                }
+               
+
+                Console.WriteLine("Creating RSAPI and Service Factory.....");
+                try
+                {
+                    
+                    _eddsDbContext = helper.GetDBContext(-1);
+                    _dbContext = helper.GetDBContext(_workspaceId);
+
+                    //create client
+                    _client = _servicesManager.GetProxy<IRSAPIClient>(ConfigurationHelper.ADMIN_USERNAME, ConfigurationHelper.DEFAULT_PASSWORD);
+
+                    Console.WriteLine("Importing Application.");
+                    if (!_debug)
+                    {
+                        //Import Application
+                        Relativity.Test.Helpers.Application.ApplicationHelpers.ImportApplication(_client, _workspaceId, true, _applicaitonFilePath, _applicationName);
+                        Console.WriteLine("Application import Complete.");
+                    }
+                    else
+                    {
+                        Console.WriteLine($"Using existing application");
+                    }
+
+                   
+
+                    _client.APIOptions.WorkspaceID = _workspaceId;
+
+                }
+                catch (Exception ex)
+                {
+                    throw new Exception("Error encountered while creating new RSAPI Client and/or Service Proxy.", ex);
+                }
+                finally
+                {
+                    Console.WriteLine("Created RSAPI and Service Factory.....");
+                }
+
+                Console.WriteLine("Creating TestObject Helper.");
+                //1 retry setting because I want to know if it fails quickly while debugging, may bump up for production
+                _testObjectHelper = new Test.Helpers.TestObjectHelper(_servicesManager, _workspaceId, 1);
+                Console.WriteLine("TestObject Helper Created.");
+
+            }
+            catch (Exception ex)
```

Comment:
Likewise, error handling and exit notification does not belong on a global level, the testing framework will handle that.

Reply:
This is all basically setup for this suite.  The main reusable parts are already in the Relativity.Test helper project.  If you think there is more that can go in there, cool.  Let's update that.  I think the logging makes it appear longer than it is.  Depending on the feedback from the above comment, I will keep or remove those according to standard practice.